backend/src/main/java/com/soen341/instagram/dto/comment/CommentResponseDTO.java

```diff
@@ -0,0 +1,65 @@
+package com.soen341.instagram.dto.comment;
```

Comment:
As mentioned in another code review, I think this can possibly benefit from having more detailed method names to specify that it is specific to the CommentResponse